integration-tests/src/test/java/gov/uk/address/api/stepdefinitions/AddressSteps.java

```diff
@@ -8,27 +8,41 @@
 import io.cucumber.java.en.And;
 import io.cucumber.java.en.Then;
 import io.cucumber.java.en.When;
+import org.junit.jupiter.api.Timeout;
+import software.amazon.awssdk.services.sqs.model.Message;
+import uk.gov.di.ipv.cri.common.library.aws.CloudFormationHelper;
+import uk.gov.di.ipv.cri.common.library.aws.SQSHelper;
 import uk.gov.di.ipv.cri.common.library.client.ClientConfigurationService;
 import uk.gov.di.ipv.cri.common.library.stepdefinitions.CriTestContext;
 
 import java.io.IOException;
 import java.text.ParseException;
+import java.util.Collections;
 import java.util.List;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
 
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertNotNull;
-import static org.junit.jupiter.api.Assertions.assertTrue;
+import static java.util.Map.entry;
+import static org.junit.jupiter.api.Assertions.*;
 
 public class AddressSteps {
-
-    private final ObjectMapper objectMapper;
+    private ObjectMapper objectMapper;
     private final AddressApiClient addressApiClient;
     private final CriTestContext testContext;
     private String uprn;
     private String postcode;
 
+    private final String auditEventQueueName =
+            CloudFormationHelper.getOutput(
+                    CloudFormationHelper.getParameter(
+                            System.getenv("STACK_NAME"), "CommonStackName"),
```

Comment:
Right so this is going to have to change if we put the mock queues into the txma infra stack too?

Reply:
Yeah that'll change after @CharlesIC PR is merged